pkg/mdm/mdmtest/apple.go

```diff
@@ -500,6 +500,26 @@ func (c *TestAppleMDMClient) AcknowledgeDeviceInformation(udid, cmdUUID, deviceN
 	return c.sendAndDecodeCommandResponse(payload)
 }
 
+func (c *TestAppleMDMClient) AcknowledgeInstalledApplicationList(udid, cmdUUID string, software []fleet.Software) (*mdm.Command, error) {
```

Comment:
my personal preference would be to keep the methods of the `mdmtest` client reduced to only the protocol operations, it's a bit more verbose for the caller, so I understand the motivation for this.